lib/auth/summarizer/summarizerv1/service.go

```diff
@@ -0,0 +1,139 @@
+package summarizerv1
+
+import (
+	"context"
+
+	pb "github.com/gravitational/teleport/api/gen/proto/go/teleport/summarizer/v1"
+	"github.com/gravitational/trace"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+// NewSummarizerService creates a new OSS version of the SummarizerService. It
+// returns a licensing error from every RPC.
+func NewSummarizerService() *ossSummarizerService {
+	return &ossSummarizerService{}
+}
+
+// ossSummarizerService is an OSS version of the SummarizerService. It returns
+// a licensing error from every RPC.
+type ossSummarizerService struct {
```

Comment:
Nit: I think "oss" prefix is redundant here. I'd just call it `summarizerService`.

```suggestion
type summarizerService struct {
```